lib/workers/repository/extract/manager-files.spec.ts

```diff
@@ -90,5 +90,51 @@ describe('workers/repository/extract/manager-files', () => {
         },
       ]);
     });
+
+    it('returns files with regex extractAllPackageFiles', async () => {
+      const managerConfig = {
+        manager: 'regex',
+        enabled: true,
+        fileList: ['airbyte.yaml'],
+        matchStrings: [
+          '# renovate: datasource=(?<datasource>[^\\s]*) depName=(?<depName>[^\\s]+)( versioning=(?<versioning>[^\\s]*))?.*\\n.*: ["]?(?<currentValue>v?[^"\\s]+)["]?\n',
+        ],
+        versioningTemplate:
+          '{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}',
+        depTypeTemplate: 'final',
+      };
+      fileMatch.getMatchingFiles.mockReturnValue(['airbyte.yaml']);
+      fs.readLocalFile.mockResolvedValueOnce(
+        '# renovate: datasource=github-releases depName=airbytehq/airbyte versioning=maven\nversion: "0.39.0-alpha"\n'
+      );
+      const res = await getManagerPackageFiles(managerConfig);
+      expect(res).toMatchSnapshot([
```

Comment:
```suggestion
      expect(res).toDeepEqual([
```

I don't think we need the snapshot here. Can we use `toDeepEqual` or are some parts are not defiuned here, so we neeed to use `toMatchObject` ?